pydantic_ai_slim/pydantic_ai/tools.py

```diff
@@ -440,12 +440,12 @@ class ToolDefinition:
     name: str
     """The name of the tool."""
 
-    description: str
-    """The description of the tool."""
-
     parameters_json_schema: ObjectJsonSchema
     """The JSON schema for the tool's parameters."""
 
+    description: str | None = None
```

Comment:
Moving this down in order for it to be able to have a default value broke keyword-less `ToolDefinition` instantiations like in the tests below. Acceptable?